src/components/form/FormFieldMessage.tsx

```diff
@@ -39,6 +39,8 @@ interface Props {
  * Any inline content.
  */
 export const FormFieldMessage = forwardRef<HTMLDivElement, Props>((props, ref) => {
+  // TODO do we need aria-live="polite" ?
```

Comment:
Yeah, it looks like we need to make the validation messages a live region if we want the validation messages to be announced by screen readers. Do you think we want screen readers to announce both success and error messages?

Reply:
Yeah I think we should, and probably the live region should be around all messages, because we will switch between error/valid/helptext